src/Cache/CacheClient.php

```diff
@@ -2279,6 +2281,89 @@ public function sortedSetLengthByScore(string $cacheName, string $sortedSetName,
         return $this->sortedSetLengthByScoreAsync($cacheName, $sortedSetName, $minScore, $maxScore, $inclusiveMin, $inclusiveMax)->wait();
     }
 
+    /**
+     * Computes the union of all the source sets and stores the result in itself. If the set does not
+     * exist, it is created with the given `ttl`. If it exists, it is overwritten with the result and
+     * its ttl is set to the given `ttl`. If the set exists but the result of the union is empty, it is deleted.
+     * The union is computed by applying the corresponding weight multiplier to the score of all elements
```

Comment:
The `destination` key doesn't actually have to be a sorted set. It could be or not be any other type; it'll be overwritten.